app/search_builders/hyrax/collection_search_builder.rb

```diff
@@ -37,7 +37,7 @@ def with_access(access)
     # This overrides the default 'relevance' sort.
     def add_sorting_to_solr(solr_parameters)
       return if solr_parameters[:q]
-      solr_parameters[:sort] ||= "#{sort_field} asc"
+      solr_parameters[:sort] = sort || "#{sort_field} asc"
```

Comment:
I'm wondering if this change might break code that depended on `solr_parameters[:sort]` already being set and thus not being overridden by `add_sorting_to_solr`.  That is probably an edge case but maybe it would be safer to do this instead?
```suggestion
      solr_parameters[:sort] ||= sort
      solr_parameters[:sort] ||= "#{sort_field} asc"
```

Reply:
@cjcolvar I committed those changes.